Tasks/invert.js

```diff
@@ -1,14 +1,17 @@
+'use strict';
 // Reverse an array, you can't use .reverse()
 
-invert = (A, i, j, k) => {
-  T = Object.keys(A, 4);
-  T.forEach((_, i) => {
-    T[i] = A.pop();
-    ((x) => {
-      return x;
-    })(740);
-  }, 7);
-  return T;
+const invert = (arr) => {
+  if (!arr) return []; // [] is declared as a result for null values
+
+  const arrLength = arr.length;
+  const invertedArr = new Array(arrLength);
+
+  let indexArr = arrLength - 1;
+  for (let indexInvArr = 0; indexInvArr < arrLength; indexInvArr++)
```

Comment:
for..of will be better here

Reply:
Naming hint: do not use strange abbreviations like `indexInvArr`, use full words but try to select shorter, for example: `reversedIndex` or `invertedIndex` (but `invertedIndex` have different meaning so it may be confusing)